### ISSUE TITLE
No more + character in environment variable WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG

### ISSUE THREAD

**Issue by MilesTEG1** [containrrr/watchtower#814]
**Describe the bug**
+ character in environment variable WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG isn't present in the mail's object since the 8 January.

**To Reproduce**
Set this environnement variable like this :
```yaml
      - WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG=[ Watchtower-DS920+ ]
```

**Expected behavior**
I expected to have the + character in my object message like this :
`[ Watchtower-DS920 ] Watchtower updates on ...`

**Screenshots**
![image](https://user-images.githubusercontent.com/29298312/107119204-64bdd180-6886-11eb-82f0-bafa961cf7f0.png)

![image](https://user-images.githubusercontent.com/29298312/107118820-941f0f00-6883-11eb-851e-19bf4f09e796.png)

**Environment**
 - Platform = Synology NAS DS920+
 - Architecture = amd64 (CPU = Intel Celeron J4125)
 - Docker version = 18.09.0-0513
 - I use Portainer CE 2.1.1 to create my container

<details>
 <summary><b> Logs from running watchtower with the <code>--debug</code> option </b></summary>

```

time="2021-02-06T14:04:43+01:00" level=debug

time="2021-02-06T14:04:44+01:00" level=debug msg="Sleeping for a second to ensure the docker api client has been properly initialized."

time="2021-02-06T14:04:45+01:00" level=debug msg="Retrieving running containers"

time="2021-02-06T14:04:45+01:00" level=debug msg="There are no additional watchtower containers"

time="2021-02-06T14:04:45+01:00" level=debug msg="Watchtower HTTP API skipped."

time="2021-02-06T14:04:45+01:00" level=info msg="Starting Watchtower and scheduling first run: 2021-02-07 06:00:00 +0100 CET"

```
</details>

**Additional context**
Before the 8 January, the + character **was present** in my notification subject (see the screenshot). But since, it's not. I didn't change the docker-compose.yml file I'm using, except for the schedule.

Here the docker-compose.yml I'm using :
```yml
#
# Doc de Watchtower : https://containrrr.dev/watchtower/
# Dépôt GitHub : https://github.com/containrrr/watchtower/
#
---
version: "2.1"
services:
  watchtower:
    image: containrrr/watchtower:amd64-latest
    container_name: watchtower
    network_mode: bridge
    environment:
      - WATCHTOWER_NOTIFICATIONS=email
      #- WATCHTOWER_NOTIFICATIONS_LEVEL=debug
      - WATCHTOWER_CLEANUP=true
      - WATCHTOWER_REMOVE_VOLUMES=true
      - WATCHTOWER_DEBUG=true
      - WATCHTOWER_LABEL_ENABLE=true
      - WATCHTOWER_TIMEOUT=30s
      # Utiliser soit SCHEDULE soit INTERVAL (ce dernier en sec)
      # Pour SCHEDULE : https://crontab.guru/#0_9_*_*_*
      # Ajouter un 0 en premier pour les secondes : secondes | minutes | heures | jour du mois | mois | jour de la semaine
      - WATCHTOWER_SCHEDULE=0 0 6 * * *
      #- WATCHTOWER_POLL_INTERVAL=3000
      - TZ=Europe/Paris
      
      #####################################################################
      # Pour les notifications en emails :
      - WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG=[ Watchtower-DS920+ ]
      - WATCHTOWER_NOTIFICATION_EMAIL_FROM=my-Gmail-email
      - WATCHTOWER_NOTIFICATION_EMAIL_TO=my-Gmail-email
      - WATCHTOWER_NOTIFICATION_EMAIL_SERVER=smtp.gmail.com
      - WATCHTOWER_NOTIFICATION_EMAIL_SERVER_PORT=587
      - WATCHTOWER_NOTIFICATION_EMAIL_SERVER_USER=my-Gmail-email
      - WATCHTOWER_NOTIFICATION_EMAIL_SERVER_PASSWORD=apasswordtomyemail
      - WATCHTOWER_NOTIFICATION_EMAIL_DELAY=2
      #####################################################################
    
    labels:
      - "com.centurylinklabs.watchtower.enable=true"
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    restart: unless-stopped


```

**Comment by piksel** [comment]
That should be fixed in shoutrrr v0.4.1 which will be in the next watchtower release.

**Comment by MilesTEG1** [comment]
Ho ! Cool :D 
I didn't do something wrong 😃
